cves/CVE-2019-3498.yml

```diff
@@ -160,8 +150,12 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    This vulnerability would require an aware user to enter a non existant
+    url path and view the returned 404 error page. As well as needing a user
```

Comment:
Grammar suggestion, connect the two - "[...] error page, as well as [..]" for complete sentences.

Reply:
Thank you, good grammar is important